index.html

```diff
@@ -1,44 +1,224 @@
+      <button type="button" class="btn">See Project</button>
+    </section>
+    <section class="card-section1">
+      <h2 class="Profesional">Profesional Art Printing Data</h2>
+      <p class="prg">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required. has been the industry's standard
+      </p>
+      <ul class="tech-list">
+        <li class="list-item">Html</li>
+        <li class="list-items">Bootstrap</li>
+        <li class="list-item">Ruby</li>
+      </ul>
+
+      <button type="button" class="btn">See Project</button>
+    </section>
+    <section class="card-section1">
+      <h2 class="Profesional">Profesional Art Printing Data</h2>
+      <p class="prg">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required. has been the industry's standard
+      </p>
+      <ul class="tech-list">
+        <li class="list-item">Html</li>
+        <li class="list-items">Bootstrap</li>
+        <li class="list-item">Ruby</li>
+      </ul>
+
+      <button type="button" class="btn">See Project</button>
+    </section>
+    <section class="card-section1">
+      <h2 class="Profesional">Profesional Art Printing Data</h2>
+      <p class="prg">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required. has been the industry's standard
+      </p>
+      <ul class="tech-list">
+        <li class="list-item">Html</li>
+        <li class="list-items">Bootstrap</li>
+        <li class="list-item">Ruby</li>
+      </ul>
+
+      <button type="button" class="btn">See Project</button>
     </section>
+  </section>
+  <section id="about" class="about-section">
+    <div class="container-2">
+      <h4 class="about-l">About me</h4>
 
+      <p class="prgs">
+        Hello I'm a software developer! I can help you build a product,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don't hestiate to
+        contact me.
+      </p>
+      <button type="submit" class="see-proj">Get My Resume</button>
+    </div>
+    <hr class="hr" />
+
+    <div class="skills">
+      <ul>
+        <li class="main">
+          <img src="./images/lang.png" alt="icon" /> Language
+        </li>
+        <li>Javascript</li>
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+      </ul>
+
+      <ul>
+        <li class="main">
+          <img src="./images/frame.png" alt="icon" /> Frameworks
+        </li>
+
+        <li>Bootstrap</li>
+
+        <li>Ruby on Rails</li>
+
+        <li>RSPec</li>
+
+        <li>CapyBara</li>
+
+        <li>Selenium</li>
+      </ul>
+
+      <ul>
+        <li class="main">
+          <img src="./images/skills-1.png" alt="circle-img" /> Skills
+        </li>
+
+        <li>Codekit</li>
+
+        <li>Github</li>
+
+        <li>Codepen</li>
+
+        <li>Github</li>
+
+        <li>Terminal</li>
+      </ul>
```

Comment:
- [ ] Kindly wrap `Languages`, `Frameworks` and `Skills` inside the same `ul` tag, not three different `ul` tags. So it will be like:
```
<ul>
     <li>Languages</li>
     <li>Frameworks</li>
     <li>Skills</li>
</ul>
```